src/styles/code.scss

```diff
@@ -136,13 +136,27 @@ code:not([class]) {
   width: 100%;
   cursor: pointer;
   position: relative;
+  box-sizing: border-box;
+  border-top: 2px solid transparent;
+  border-bottom: 2px solid transparent;
+}
+
+.highlight-copy-block:focus {
+  border-top: 2px solid var(--amplify-colors-neutral-10);
+  border-bottom: 2px solid var(--amplify-colors-neutral-10);
+
+  @include darkMode {
+    border-top: 2px solid var(--amplify-colors-neutral-90);
+    border-bottom: 2px solid var(--amplify-colors-neutral-90);
+  }
 }
 
 .highlight-copy-block-hint {
   position: absolute;
   top: 0;
   right: 1.8rem;
   color: white;
+  user-select: none;
```

Comment:
When I test this locally, it still copies "highlighted code example" since that text exists outside of this class. You can test by using your mouse to drag and copy/paste text.

Reply:
I tried to select it (posted a screenshot of that on the PR overview). Can you screenshot your case, maybe I'm misinterpreting the issue.

Reply:
Yeah, I'm testing on this page locally with the code block in this section: http://localhost:3000/react/build-a-backend/add-aws-services/deletion-backup-resources/#enabling-point-in-time-recovery-for-dynamodb-tables

Drag your mouse to select the whole code block, then use ctrl + c or right click to copy and paste the content somewhere. At the end of what I paste, I see the text "highlighted code example" because that text exists outside of the class you added user-select: none to:

<img width="526" alt="Screenshot 2024-05-07 at 2 36 05 PM" src="https://github.com/aws-amplify/docs/assets/376920/5162f12e-32e7-4f76-a673-36d6d396a771">